### ISSUE TITLE
while parsing the marker pack, i found very simple issues

### ISSUE THREAD

**Issue by coderedart** [xrandox/TehsTrails#21]
at line 1692,
```xml
<POI MapID="19" xpos=" -54.0497" ypos="43.6899" zpos="-263.195" type="tt.mc.cm.rm.waypoints" GUID="nrTgPIqKEkOWLj54XtTtCQ=="/>
```
there's a space between `xpos="` and `-54` which made the parsing the whole file fail xD. such a sneaky little silly thing. 

the iconFile tag attribute refers to this paths. but the case of the image names is wrong. eg: Vista.png is present as vista.png in the folder.
/Data/MapCompletion/General/Vista.png,
/Data/MapCompletion/WaypointMarkers/WPHeart.png
/Data/MapCompletion/General/heartspecific/Threaten.png,
/Data/MapCompletion/General/heartspecific/Cheer.png,
/Data/MapCompletion/General/heartspecific/Bow.png, 

both of these categories are referred by POI tags, but these categories do not exist. a simple ctrl +f should help you find this
tt.mc.cm.rm.ppr_m
tt.mc.cm.rm.ppr_r

both of these trails exist, but their name has a small case of `O` in the TrailOfShame.trl part. 
 Data/MapCompletion/Trails/trailsofshame/LornarsPassTrailOfShame.trl,
Data/MapCompletion/Trails/trailsofshame/FrostgorgeSoundTrailOfShame.trl,

and finally, The trail tags don't have a guid. i made the parser generate new guids like taco/blish, but if you are fine with it, i would love for guid to be a required attribute.

**Comment by dlamkins** [comment]
@coderedart I see this one is a bit old but:
- The paths are case insensitive.  The VFS you create from the ZIP should be able to easily account for this.
- Trail tags don't use GUID attributes because there aren't any features in TacO or Blish HUD where a trail could be affected by a behavior directly.

I'm surprised whatever your float parse method is-is sensitive to the space.  Most I've seen trim or otherwise discard the extra space.

**Comment by coderedart** [comment]
yeah, 5 months is a long time. I have changed like 3 xml parsers in the mean time, and now i don't even use serde and manually deserialize marker pack from the dom tree to deal with all these issues.
1. case sensitivity. I did ask tekkit a couple days ago and he mentioned categories are case sensitive in taco. but it seems paths are case-insensitive because windows is. i'm normalizing everything to lower case now. 
2. I made all attributes optional except "type" and "MapID", so no need for GUIDs anymore either.
3. I now trim the white space myself. but serde gives the value directly to float parser before trimming it seems. no more serde deserializing the xml packs, so this is fine too. 

I will just close the issue as all of these are not really a problem for me anymore.
